index.js

```diff
@@ -1,6 +1,43 @@
+const projects = {
```

Comment:
- Implement an array of objects to store JavaScript objects containing the information for each project.